### ISSUE TITLE
3 momentum in `convert_to_hist_list`

### ISSUE THREAD

**Issue by mhostert** [KlingFelix/FORESEE#46]
Is this supposed to be have pz**2 instead of pz? 

https://github.com/KlingFelix/FORESEE/blob/02a6d10ec2b4665d3f224dad3230a095d16e9729/src/foresee.py#L427C29-L427C52

**Comment by shivasankarka** [comment]
I believe it should be be pz**2 since it's the total momenta.

**Comment by KlingFelix** [comment]
Yes. I think this has been fixes (at leats in the dev branch). Fortunately, it looks like this particular function (so with a 4-dimensional array as input) is not called anywhere inside the code. This would explain why I haven't noticed it before.
